### PR TITLE
test: Refactor to use call_method()

### PR DESCRIPTION
**Pull request by martinpitt**
Introduced in commit 3e540a0e167e.

### CODE REVIEW

**Comment by martinpitt** [comment]
Ah, one more piece of cleanup

File: test/test_basic.py [hunk 1]
```diff
@@ -71,133 +62,117 @@ async def _call():
 
     def test_noarg_noret_sync(self):
         self.add_method('', 'Do', '', '', '')
-        message = self.build_call('Do')
-        self.assertEqual(self.bus_user.call(message).get_body(), ())
+        result = self.bus_user.call_method(N_TEST, '/', I_TEST, 'Do')
```
Comment (allisonkarlitskaya) [inline test/test_basic.py:65]: I actually sort of liked that your wrapper filled this in a bit.  I wonder if we could figure out some way to save this stuff using a partial application or so...

Alternatively, maybe you could make it "nicer" by doing something evil like

```
TEST_ADDR = ['org.freedesktop.Test', '/', 'org.freedesktop.Test.Main']
```

and then

```
self.bus_user.call_method(*TEST_ADDR, 'Do')
```

**Review by allisonkarlitskaya — COMMENTED** [review]
I'm actually going to tweak the `.call_method()` functions to return the body directly.  Please stand by.

File: test/test_basic.py (hunk 1 above)
Comment (martinpitt) [reply test/test_basic.py]: You are right, this looks nicer. How about now?

Comment (allisonkarlitskaya) [reply test/test_basic.py]: :+1:

**Review by allisonkarlitskaya — APPROVED** [review]
A lot nicer now, thanks :)

**Pull request merged**